### PR TITLE
In 0 decrease ansible execution time

### PR DESCRIPTION
**Pull request by lmesz-bitrise**
### New Pull Request Checklist

*Please mark the points which you did / accept.*

- [ ] The tool I added is stable, and __does not require frequent updates__. _Preinstalled tools on the LTS stacks
  are not updated, so if the tool requires frequent updates you should handle the installation on-demand (see: [Install Any Additional Tool - DevCenter](https://bitrise-io.github.io/devcenter/tips-and-tricks/install-additional-tools/) for more information)_
- [ ] I added a version report line to [`system_report.sh`](/system_report.sh) for the new tool(s) I added
- [ ] I have added an entry to the CHANGELOG.md with a revisionID (the current date and the number of the change e.g.: 2019_10_11_1) and a brief description of the actual change
- [ ] I have updated the revisionID in [bitrise_profile](roles/profiles/files/bitrise_profile) **BITRISE_OSX_STACK_REV_ID** with the one I just created in the CHANGELOG.md

### CODE REVIEW

File: roles/tmp-setup/tasks/main.yml [hunk 1]
```diff
@@ -68,6 +68,12 @@
   shell: bash -l -c "rbenv global {{ default_ruby }}"
   when: is_incremental_setup|default(false) == false
 
+# visual studio
```
Comment (fehersanyi-bitrise) [inline roles/tmp-setup/tasks/main.yml:71]: I think this is not part of this PR :)

File: bitrise.yml [hunk 2]
```diff
@@ -149,7 +149,7 @@ workflows:
           inputs:
             - content: |-
                 #!/usr/bin/env bash
-                echo -e "[defaults]\ncallback_whitelist=profile_tasks" > ansible.cfg
+                echo -e "[defaults]\ncallback_whitelist=profile_tasks\n[ssh_connection]\npipeline=True" > ansible.cfg
```
Comment (fehersanyi-bitrise) [inline bitrise.yml:152]: how will this affect the runtime?

File: roles/tmp-setup/tasks/main.yml (hunk 1 above)
Comment (lmesz-bitrise) [reply roles/tmp-setup/tasks/main.yml]: Oooooo you are right, sorry removed
